my_internet_speed/settings.py

```diff
@@ -19,12 +19,13 @@
 
 CELERY_CONFIG = {'broker': REDIS_URL, 'timezone': TIMEZONE}
 
-TWITTER = Api(
-    consumer_key=config('TWITTER_CONSUMER_KEY'),
-    consumer_secret=config('TWITTER_CONSUMER_SECRET'),
-    access_token_key=config('TWITTER_ACCESS_TOKEN'),
-    access_token_secret=config('TWITTER_ACCESS_TOKEN_SECRET')
-)
+CREDENTIALS = {
+        'consumer_key': config('TWITTER_CONSUMER_KEY', default=None),
+        'consumer_secret': config('TWITTER_CONSUMER_SECRET', default=None),
+        'access_token': config('TWITTER_ACCESS_TOKEN', default=None),
+        'access_token_secret': config('TWITTER_ACCESS_TOKEN_SECRET', default=None)
+        }
```

Comment:
@nicolastarzia, would you mind some stylistic changes here? Lines 23-27 are improperly indented. 

Lines 23-26 should have 4 spaces instead of 8.
Line 27 should have no indentation.